src/main/scala/org/coroutines/Coroutine.scala

```diff
@@ -172,10 +172,26 @@ object Coroutine {
       $yield
     }
 
+    /** Returns whether or not the coroutine yielded a value. This value can be
+     *  accessed via `getValue`.
+     *
+     *  @return `true` if the coroutine yielded a value, `false` otherwise.
+     */
     final def hasValue: Boolean = $hasYield
 
+    /** Returns an `Option` instance wrapping the current value of the coroutine,
```

Comment:
Unless I'm looking at the wrong diff, I only see one sentence in this description. Did you want me to separate a different part of the doc?